scripts/validate_docstrings.py

```diff
@@ -471,6 +471,43 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        i = list(self.doc_parameters).index(param)
+        desc_list = [line
+                     for line in self.doc._parsed_data['Parameters'][i][-1]
+                     if line]
+        # Find and strip out any sphinx directives
+        for directive in DIRECTIVES:
```

Comment:
Can you convert this to a regex? I think simple enough and would be much easier to read than existing loops

Reply:
Hi @WillAyd ,
Thank you for having a look at this. I have replaced this with a regex, not sure if this is what you had in mind. Please let me know if I have to change something.